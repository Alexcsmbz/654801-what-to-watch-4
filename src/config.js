export const moviePageTabs = [
  {
    name: `Overview`,
  },
  {
    name: `Details`,
  },
  {
    name: `Reviews`,
  },
];

export const RoutePath = {
  MAIN: `/`,
  LOGIN: `/login`,
  MOVIE: `/films/:id`,
  MYLIST: `/mylist`,
  REVIEW: `/films/:id/review`,
};
