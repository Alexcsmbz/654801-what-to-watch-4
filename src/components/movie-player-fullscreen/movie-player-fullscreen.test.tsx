import MoviePlayerFullscreen from './movie-player-fullscreen';
import {testMovie} from 'config';

describe(`MoviePlayerFullscreen snapshot test`, () => {
  it(`MoviePlayerFullscreen should render fullscreen movie player`, () => {
    const tree = renderer.create(
        <MoviePlayerFullscreen
          setIsFullscreen={() => {}}
          movie={testMovie}
          isFullscreen={true}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
