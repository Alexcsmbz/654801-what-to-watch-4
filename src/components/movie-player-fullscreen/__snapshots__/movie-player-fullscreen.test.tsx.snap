// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePlayerFullscreen snapshot test MoviePlayerFullscreen should render fullscreen movie player 1`] = `
<video
  controls={true}
  height="175"
  width="280"
>
  <source
    src="videoLink"
    type="video/mp4"
  />
</video>
`;
